roles/eda/templates/eda.configmap.yaml.j2

```diff
@@ -11,6 +11,9 @@ data:
   # Operator specific settings
   EDA_DEPLOYMENT_TYPE: "k8s"
 
+  # Public Hostname
+  EDA_WEBHOOK_BASE_URL: "{{ public_hostname }}"
```

Comment:
@msmagnanijr this must be "http[s]://{{ public_hostname }}{{ eda_webhook_prefix_path }}"

@rooftopcellist I wonder if it can be `public_base_url` so it can include the scheme and we don't have to figure out it nor assume it is https.